### PR TITLE
sync system-profile

### PR DESCRIPTION
**Pull request by tahmidefaz**
## Overview

Syncing System Profile. PR: https://github.com/RedHatInsights/inventory-schemas/pull/43

This is a copy-paste of the schema.

### CODE REVIEW

**Review by kruai — APPROVED** [review]

File: swagger/system_profile.spec.yaml [hunk 1]
```diff
@@ -267,7 +267,7 @@ $defs:
           type: string
           maxLength: 512
       installed_packages_delta:
-        type: array # sorted list, rest of packages not in installed_packages
+        type: array  # sorted list, rest of packages not in installed_packages
```
Comment (kruai) [inline swagger/system_profile.spec.yaml:270]: Was this change intentional @tahmidefaz ?

Comment (tahmidefaz) [reply swagger/system_profile.spec.yaml:270]: Yes, I think this wasn't copied over earlier

Comment (kruai) [reply swagger/system_profile.spec.yaml:270]: OK. Was a minor detail but wanted to make sure it was intentional. Still LGTM

Comment (tahmidefaz) [reply swagger/system_profile.spec.yaml:270]: Yes, I wasn't sure when I was opening the PR. I had to double-check also :+1:

**Review by ryandillinfelton — APPROVED** [review]
LGTM

**Comment by kruai** [comment]
Gonna merge this bc the corresponding inventory-schemas PR got merged
